src/scala/tiefvision-web/app/core/ImageProcessing.scala

```diff
@@ -24,8 +24,8 @@ object ImageProcessing {
   def randomSmilarityImage = SimilarityImages(random.nextInt(SimilarityImages.length))
 
   def findSimilarImages(image: String, finderFolder: String, imagesFolderOpt: Option[String] = None): ImageSearchResult = {
-    val imagesFolderOptionOpt = imagesFolderOpt.map(imagesFolder => s"-imagesFolder ${imagesFolder}").getOrElse("")
-    val finderProcessBuilder = Process(Seq("bash", "-c", s"luajit search.lua $imagesFolderOptionOpt -image $image"), new File(finderFolder))
+    val luaSearch = s"luajit search.lua $image ${imagesFolderOpt.getOrElse("")}"
```

Comment:
A bit puzzled as to why `imagesFolderOpt` is an optional. From what I understand, we're searching for an image in a folder here, so it's empty, what's gonna happen?